src/ir_Sharp.cpp

```diff
@@ -66,15 +66,24 @@ void IRsend::sendSharpRaw(uint64_t data, uint16_t nbits, uint16_t repeat) {
     // Note: Previously this used to be performed 3 times (normal, inverted,
     //       normal), however all data points to that being incorrect.
     for (uint8_t n = 0; n < 2; n++) {
+#if SEND_DENON
+	    // comment out #define	SEND_SHARP in IRremoteESP8266.h file!!
```

Comment:
We can't allow the situation where a user can only have Sharp or Denon, not both.
Plus, based on your comments in the PR, I'm not sure you're using the data correctly